spec/powershell-spec.coffee

```diff
@@ -111,28 +103,29 @@ describe "PowerShell grammar", ->
 
       it "should tokenize variable names", ->
         expect(tokens[3].value).toEqual "name"
-        expect(tokens[3]).toHaveScopes ["embedded.variable.other.powershell"]
+        expect(tokens[3]).toHaveScopes ["source.powershell", "string.quoted.double.powershell", "variable.other.readwrite.powershell"]
 
       it "should not tokenize as a variable when leading $ has been escaped", ->
         expect(tokens[5].value).toEqual "`$"
-        expect(tokens[5]).toHaveScopes ["source.powershell", "string.quoted.double.single-line.powershell", "constant.character.escape.powershell"]
-        expect(tokens[5]).not.toHaveScopes ["embedded.variable.other.powershell"]
+        expect(tokens[5]).toHaveScopes ["source.powershell", "string.quoted.double.powershell", "constant.character.escape.powershell"]
+        expect(tokens[5]).not.toHaveScopes ["source.powershell", "embedded.variable.other.powershell"]
 
         expect(tokens[6].value).toEqual "bob"
-        expect(tokens[6]).not.toHaveScopes ["embedded.variable.other.powershell"]
+        expect(tokens[6]).not.toHaveScopes ["source.powershell", "embedded.variable.other.powershell"]
 
   describe "Keywords", ->
     describe "Block keywords", ->
+      # TODO: Add "sequence", "workflow" to keywords
```

Comment:
`workflow` is the same as `function` in our current grammar https://github.com/SublimeText/PowerShell/blob/master/Support/PowershellSyntax.tmLanguage#L525 for sequence I opened https://github.com/SublimeText/PowerShell/issues/105